src/subscript/fmu_copy_revision/fmu_copy_revision.py

```diff
@@ -0,0 +1,655 @@
+#!/usr/bin/env python
+#
+# fmu_copy_revision, for fast and secure copy for fmu revisions
+# using rsync as engine, with xargs to speed up multithreading
+#
+import sys
+from os.path import join
+import argparse
+import pathlib
+import tempfile
+import time
+import getpass
+import subprocess
+import shutil
+from multiprocessing import cpu_count
```

Comment:
Can you do `pip install isort; isort fmu_copy_revision.py`  ?

Reply:
Done!